content/en/docs/faas/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Cloud Functions
```

Comment:
For consistency call this "Functions as a Service", if this is to long for the sidebar menu you still can have a linkTitle like "Cloud Functions" or "FaaS"